### ISSUE TITLE
About the BatchNorm

### ISSUE THREAD

**Issue by Solacex** [hendrycks/anomaly-seg#6]
<img width="631" alt="WeChate11346ede9ca427e9df392f5438d6dee" src="https://user-images.githubusercontent.com/12868455/82635719-3179da80-9c44-11ea-9435-0f486c5da133.png">
Firstly, thanks for your inspiring work~!

In your paper,  you mention that the BatchNorm is frozen, but I notice that your implementation doesn't freeze the BN. So I want to know if there are any mistakes or my misunderstanding.

**Comment by Solacex** [comment]
And, could you share the model trained on StreetHazards? I only have limited GPUs.

**Comment by xksteven** [comment]
Thanks for your comments about batchnorm.  We are in the process of updating the paper.

Also here's a copy of the ppm model with deepsupervision weights that we used during evaluation.
https://drive.google.com/file/d/1gaaBMY42HiQ9SBQ9N8Rv258HuCvVox8u/view?usp=sharing
